src/exception.rs

```diff
@@ -187,6 +192,30 @@ fn handle_data_abort(context_frame: &mut TrapFrame) -> AxResult<AxVCpuExitReason
     })
 }
 
+/// Handles HVC exceptions that serve as psci (Power State Coordination Interface) calls.
+///
+/// Returns `None` if the HVC is not a psci call.
+fn handle_psci_hvc(ctx: &mut TrapFrame) -> Option<AxResult<AxVCpuExitReason>> {
```

Comment:
handle_psci

Both `smc` and `hvc` handlers can call this function

Reply:
OK

Reply:
Added